src/crypto-api.ts

```diff
@@ -713,7 +713,8 @@ export interface CrossSigningKeyInfo {
  * Recovery key created by {@link CryptoApi#createRecoveryKeyFromPassphrase}
  */
 export interface GeneratedSecretStorageKey {
-    keyInfo?: AddSecretStorageKeyOpts;
+    /** Information to generate the key from a passphrase if any. */
```

Comment:
Nit: could be a bit clearer here

```suggestion
    /** If the key was derived from a passphrase, information (algorithm, salt, etc) on that derivation. */
```